programs/eosio-launcher/main.cpp

```diff
@@ -605,8 +615,17 @@ launcher_def::initialize (const variables_map &vmap) {
     }
   }
 
-  config_dir_base = "etc/eosio";
-  data_dir_base = "var/lib";
+  config_dir_base_cluster0 = "etc/eosio";
+  if (cluster_id == 0) {
+    config_dir_base = config_dir_base_cluster0;
```

Comment:
Why not be consistent and do the same thing as when `cluster_id != 0`? 

So in this case `config_dir_base` could be the path "etc/eosio/cluster00000" and `data_dir_base` could be the path "var/lib/cluster00000".